### PR TITLE
DDF-1200 - Added positive test case for fanout

### PR DESCRIPTION
**Pull request by mverret**
Also updated setFanout() method to accept a boolean so you can turn fanout on and off.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/39)

### CODE REVIEW

**Comment by mverret** [comment]
@shaundmorris

**Comment by shaundmorris** [comment]
LGTM. Verified tests pass. Nice to have some fanout Integration tests now.

**Comment by shaundmorris** [comment]
Planning on merging tomorrow 5/28

File: test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java [hunk 1]
```diff
@@ -287,17 +292,56 @@ public void testOpensearchToCswSourceToCswEndpointQuerywithMetacardXml() throws
 
 
     @Test
-    public void testFanout() {
+    public void testFanoutQueryAgainstUnknownSource() {
         try {
-            setFanout();
+            setFanout(true);
+            waitForAllBundles();
+        } catch (IOException e) {
```
Comment (shaundmorris) [inline test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java]: @mverret you can just do (IOException | InterruptedException e) to make it a little cleaner

**Comment by shaundmorris** [comment]
Verified build and tests passing.

**Pull request merged**